### ISSUE TITLE
Home: Incorrect Labelling Pattern for Search Input

### ISSUE THREAD

**Issue by jscholes** [benetech/ImageShare#54]
There must be a persistently visible label directly above or to the left of the main search input, with text such as “Search Resources” or similar.  In addition, the placeholder text must be replaced with inline text directly on the page below the field, linked to the `<input>` via `aria-describedby`.  The current placeholder text, "Enter name of collection or resource to search (try “geology” or “heat”)", supplies instructions as well as examples, so it is not appropriate for it to be rendered in this way.

![Screenshot of the search input with no visual label, placeholder, Enter name of collection or resource to search (try “geology” or “heat”).](https://user-images.githubusercontent.com/6454755/80000025-3b13f500-8482-11ea-85aa-4c54c093945f.png)

**Comment by jscholes** [comment]
Additional note: as displayed in the current design materials, the placeholder text only has a color contrast ratio of 2.8:1 (below the 4.5:1 minimum requirement).  This should be resolved once the text is moved to be rendered inline on the page, as placeholders can never be guaranteed to have sufficient readability.

**Comment by sinabahram** [comment]
Can @jkva  or @vincent-young-1701  please confirm this is handled now in new designs/implementation? If so, please close this  issue.

**Comment by vincent-young-1701** [comment]
This has been taken care of in the new.  Closing.
